superset-frontend/src/explore/components/controls/DatasourceControl/index.jsx

```diff
@@ -117,6 +118,15 @@ class DatasourceControl extends React.PureComponent {
 
   onDatasourceSave(datasource) {
     this.props.actions.setDatasource(datasource);
+    // remove time column in the form_data
+    const timeCol = this.props.form_data?.granularity_sqla; // eslint-disable-line camelcase
```

Comment:
As there are multiple references to snake case variables, maybe we could just disable it for the whole file:
```js
/* eslint-disable-line camelcase */
```